ros/api/v1/hosts.py

```diff
@@ -184,12 +170,12 @@ def build_system_filters():
             modified_states = []
             for state in states:
                 state = state.capitalize()
-                if state not in SYSTEM_STATES_EXCEPT_EMPTY:
+                if state not in SystemStates.SYSTEM_STATES_EXCEPT_EMPTY.value:
```

Comment:
We can add ``SystemStatesWithKeys.SYSTEM_STATES.value.values()`` here as suggested in ``constants.py`` file